src/app/App.java

```diff
@@ -197,6 +229,19 @@ public Collection collection_add(Collection collection, Game game) {
         return null;
     }
 
+    /**
+     * Removes a game from a user's collection. Game and collection are confirmed
+     * as valid prior to the call of the function.
```

Comment:
why are we assuming that these are valid before calling the function? makes more sense to do validation here than in cli/frontend